synology_dsm/api/storage/storage.py

```diff
@@ -0,0 +1,156 @@
+"""DSM Storage data."""
+# -*- coding:utf-8 -*-
+from synology_dsm.helpers import SynoFormatHelper
+
+
+class SynoStorage(object):
+    """Class containing Storage data."""
+
+    def __init__(self, raw_data):
+        self._data = {}
+        self.update(raw_data)
+
+    def update(self, raw_data):
+        """Updates storage data."""
+        if raw_data:
+            self._data = raw_data["data"]
+
+    @property
+    def volumes(self):
+        """Returns all available volumes."""
+        volumes = []
+        for volume in self._data.get("volumes"):
+            volumes.append(volume["id"])
+        return volumes
+
+    def _get_volume(self, volume_id):
+        """Returns a specific volume."""
+        for volume in self._data.get("volumes"):
+            if volume["id"] == volume_id:
+                return volume
+        return {}

```

Comment:
Maybe return `None` here, getting an error when doing `.get()` on this may be wanted.

Reply:
No error handling for now